microsoft-edge/extensions-chromium/store-policies/ada-addendum.md

```diff
@@ -13,22 +13,28 @@ ms.date: 02/17/2021
 <!-- ====================================================================== -->
```

Comment:
Maybe change the title of this article from
App Developer Agreement Addendum for Microsoft Edge program users
to
Addendum to Microsoft Edge Add-ons store developer policies

and swap the TOC order of the two docs to become:
Microsoft Edge Add-ons store developer policies
Addendum to Microsoft Edge Add-ons store developer policies

Reply:
I believe that the link destination is incorrect.  The present article is not an addendum to the store policies, but to the ADA.  All evidence is consistent with that except the actual link destination.  I'm entering a Suggestion to fix the link presentation & destination.